build/mock-rbac/Dockerfile

```
@@ -0,0 +1,34 @@
+##
+# This build the container image for the service
+##
+FROM registry.access.redhat.com/ubi9/go-toolset:1.20 as builder
+LABEL idmsvc-backend=builder
+# https://developers.redhat.com/articles/2022/05/31/your-go-application-fips-compliant
+ENV OPENSSL_FORCE_FIPS_MODE=1
+WORKDIR /go/src/app
+COPY . .
+USER 0
+RUN git log -1
+RUN make get-deps bin/mock-rbac
```

Do we need `get-deps`? Does the container build work without it?
```suggestion
RUN make bin/mock-rbac
```